### DISCUSSION TITLE
How do you deal with the bounding box?

### DISCUSSION THREAD

**Discussion by AnnaMei11** [assimp/assimp#4396] [Q&A] [answered]
Hello! I'm trying to get the sample script 3d_view_py3.py script to work, but I can't because the bounding box always reads my incoming matrix as a singular matrix that can't be inverted and therefore doesn't work with the helper.py script where the get bounding box function is described. The script will run if I use dummy values for the bounding box but then a I just get a blank screen with no model. I've tried this with multiple models wit multiple different file extensions. What am I missing? Is it the bounding box, or is there something else wrong with that script?

Here is the code I injected for the dummy values. Other than that, let me know what details I can give that would be helpful!
```
self.bb_min =[-3,-3,-3]
       
self.bb_max=[3,3,3]
```

**Comment by kimkulling** [comment, marked as answer]
Hi, 
I am wondering why you want to define your bounding-box by yourself. The default usecase is to get the bounding box from the imported model and transform it afterwards. Normally all these things will be done automaticall when you asset was loaded correctly. 

So from my point of view there is no need to define youe bounding box by yourself.

If you want to do it would be really useful to understand the full usecase you have to adress.

Kim
